test/chunking-form/samples/mixed-synthetic-named-exports-2/_expected/es/generated-dep2.js

```diff
@@ -0,0 +1,3 @@
+var dep2 = {bar: {foo: 'works'}};
+
+export default dep2;
```

Comment:
I do not think this should be rendered as a default export. Usually, there are no default exports in auto-generated chunks and everything is a named export. Furthermore, the named exports are also minified. I think we definitely need this here as well, the most important reason being that a file can have only one default export while a chunk could potentially have many of those synthetic exports belonging to different default exports. I think this bug is also shown in the test case I added in the other comment.

Reply:
you right! I just noticed rollup never emits default exports in generated chunks